R/getPrevalence.R

```diff
@@ -275,7 +275,7 @@ setMethod("getPrevalence", signature = c(x = "SummarizedExperiment"),
 
         # check assay
         .check_assay_present(assay.type, x)
-        x <- .agg_for_prevalence(x, rank = rank, ...)
+        x <- .agg_for_prevalence(x, agg.na.rm = FALSE, rank = rank, ...)
```

Comment:
This disables the usage of agg.na.rm. I reverted it

Reply:
ACK.